src/ProfilesList.json

```
@@ -218,5 +218,9 @@
   "g0v1ndN.json",
   "Sairajepawar.json",
   "Haris-Mohanty.json",
+  "THE-AryanKumar.json",
+  "leabuende.json",
+  "destryptor.json",
   "leabuende.json"
+  "rishabh22204901"
```

```suggestion
  "rishabh22204901.json"
```
Add complete filename that is `rishabh22204901.json`